tests/workforce-allocations-to-delius/allocate-new-person.spec.ts

```diff
@@ -74,6 +77,7 @@ test('Allocate previously-managed person', async ({ page }) => {
     // And a new unallocated event and requirement
     await createCommunityEvent(page, { crn, allocation: data.teams.allocationsTestTeam })
     await createRequirementForEvent(page, { crn, eventNumber: 2, team: data.teams.allocationsTestTeam })
+    await createInitialAppointment(page, crn, '2')
```

Comment:
Do we also need to check that the initial appointment date appears correctly in the allocations service?